snippets/ponylang-mode/be

```diff
@@ -3,5 +3,5 @@
 # name: be
 # contributor: Sean T. Allen <sean@monkeysnatchbanana.com>
 # --
-be ${1:name}(${2:params}) =>
+be ${1:name}(${2:params}: ${3: type}) =>
```

Comment:
As return type is options, I'm not so sure about this.

Reply:
it's a copy error, i can fix it